blueocean-dashboard/src/main/js/components/RunDetailsHeader.jsx

```diff
@@ -109,6 +104,8 @@ class RunDetailsHeader extends Component {
     }
 }
 
+const { object, func } = PropTypes;
```

Comment:
Again, much opposed to setting variables up for almost no reason, when you can just reference `PropTypes.thing`. Also, unsure the validity but there has been comments this breaks `flow`'s ability to typecheck certain things.